src/views/components/model.jsx

```diff
@@ -22,6 +22,10 @@ export default class Model extends React.Component {
         }
     }
 
+    deleteModel() {
+        ModelController.deleteModel(this.state.model.address);
```

Comment:
I already added a pbi for this the last time you mentioned it. https://msoese.atlassian.net/browse/TFF-121